dashboard/app/helpers/notes_helper.rb

```diff
@@ -3,13 +3,24 @@ module NotesHelper
 
   def get_slides_by_video_key(video_key)
     slides_key = "slides.#{video_key}"
-    slides = try_t(slides_key)
-    return nil unless slides
+    slides = {}
+    1.step.each do |n|
+      slide_text = try_t("#{slides_key}.#{n}.text")
+      slide_image = try_t("#{slides_key}.#{n}.image")
+      break unless slide_text
+      slides[n] = {text: slide_text, image: slide_image}
+    end
+    return nil if slides.empty?
     return slides if I18n.locale == I18n.default_locale
 
     # If we're in a non-english locale, fold in the English yaml's slides for
     # image references to avoid previously translation-broken asset paths
-    slides_english = try_t(slides_key, locale: :en)
+    slides_english = {}
+    1.step.each do |n|
+      slide_image = try_t("#{slides_key}.#{n}.image", locale: :en)
+      break unless slide_image
+      slides_english[n] = {image: slide_image}
+    end
```

Comment:
Assuming `I18n.default_locale` is english, the above `return slides` would be invoked if `I18n.locale` is also english, making the comment accurate (since this line is only reached if we're in a non-default [english] locale).

Reply:
👍  I think that line was excluded from the original diff. Or maybe I just missed it